lib/std/io/reader.zig

```diff
@@ -144,12 +144,13 @@ pub fn Reader(
         pub fn readUntilDelimiter(self: Self, buf: []u8, delimiter: u8) ![]u8 {
```

Comment:
The documentation should be updated to say something like "The delimiter byte is written to the output buffer, but is not included in the returned slice."

Reply:
Oh yes, definitely! I'm only waiting until we have a resolute "new behavior" so I can update all those docs ;-)